mbed_lstools/lstools_base.py

```diff
@@ -134,7 +130,8 @@ def list_mbeds(
             if  ((not device['mount_point'] or
                   not self.mount_point_ready(device['mount_point'])) and
                  not self.list_unmounted):
-                if  (device['target_id_usb_id'] and device['serial_port']):
+                if  (device['target_id_usb_id'] and device['serial_port']
+                    and self.__class__.__name__ != 'MbedLsToolsWin7'):
```

Comment:
Is there a way to do this without using dunderscore (`__`) variables?

Reply:
Yeah I can do it another way, I'll revise in a bit.